src/main/java/io/appform/dropwizard/actors/actor/UnmanagedBaseActor.java

```diff
@@ -174,4 +185,13 @@ public String visit(DefaultConnectionStrategy strategy) {
         });
     }
 
+    private void validateActorConfig(ActorConfig actorConfig) {
```

Comment:
This can be added as a validationMethod in the config class itself

Reply:
Do you suggest moving the method to ActorConfig, but having the invocation in UnmanagedBaseActor. (as it is right now.)?

Reply:
done.